api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -71,6 +71,23 @@ message AccessListSpec {
 
   // owner_grants describes the access granted by owners to this access list.
   AccessListGrants owner_grants = 11;
+
+  // member_access_lists is a list of access lists that user
+  // membership should be fetched from
```

Comment:
So exclusively user memberships are pulled, and nothing else? I think it's worth being extra explicit in the language here.

Are the membership_requires of the parent AccessList applied before calculating the members?

Are the membership_requires of the present AccessList further applied to the members of the "members lists"?

Please document.